cypress/downloads/response.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Looks like this response file will be created once you've ran the code. You can remove them from the commit